scripts/linux/buildManifest.sh

```diff
@@ -156,14 +154,13 @@ sed "s/__REGISTRY__/${DOCKER_REGISTRY}/g; s/__VERSION__/${DOCKER_IMAGEVERSION}/g
 echo "Build image with following manifest:"
 cat $manifest
 
-echo "Done Building And Pushing Docker Images"
+# Download and execute the manifest tool
+curl -Lo /tmp/manifest-tool 'https://edgebuild.blob.core.windows.net/iotedge-win-arm32v7-tools/manifest-tool-linux-amd64' &&
+    chmod +x /tmp/manifest-tool &&
+    /tmp/manifest-tool --debug push from-spec $IGNORE_MISSING $manifest
 
+echo "Done Building And Pushing Docker Images"
```

Comment:
Thanks for pushing down this log statement. I thought it was odd the log was emitted _before_ the tool was invoked. That may be part of what led to the confusion... 👍

Reply:
yep